.travis.yml

```diff
@@ -1,7 +1,7 @@
-language: node_js
-
 sudo: false
```

Comment:
I would strongly prefer to do this in a github action. Travis is moving to paid plans, as far as I understood.

Reply:
Yeah cool! I started on this patch before travis did the announcement and picked travis because this is where nok releases are done. I can port this to github actions for sure.

Reply:
Please do, as far as I remember the travis stuff isn't working anymore anyway. So I'm probably going to remove that at some point.